### PR TITLE
Enforce hostgroup policies

### PR DESCRIPTION
**Pull request by shlomizadok**
Make sure that hostgroup policies are also applied to host policies.

### CODE REVIEW

**Comment by shlomizadok** [comment]
(other tests are failing due to change in api uri, fixing tests at: https://github.com/shlomizadok/foreman_openscap/tree/host_form_scap_policy_proxy/test)

**Comment by shlomizadok** [comment]
@ares @isimluk - ping... can you test please?

File: test/unit/openscap_host_test.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Setting[:token_duration] = 0
+    Scaptimony::Policy.any_instance.stubs(:ensure_needed_puppetclasses).returns(true)
+  end
+
+  test 'Host has policy' do
+    host = FactoryGirl.create(:host)
+    assert_empty(host.policies)
+    policy = FactoryGirl.create(:policy)
+
+    assert(policy.assign_hosts([host]), 'Host policies should be assigned')
```
Comment (ares) [inline test/unit/openscap_host_test.rb]: would it make sense to check that host has really the policy assigned?

**Comment by ares** [comment]
Tested, except the test nit it looks good and works nicely.

**Comment by shlomizadok** [comment]
@ares Thanks for reviewing. Added the test.

File: test/unit/openscap_host_test.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    assert(host.policies.include?(policy), 'Policy should be in Host policies')
```
Comment (ares) [inline test/unit/openscap_host_test.rb]: could you please use `assert_includes(collection, expected)`, it's more verbose and you don't have to specify custom message

**Comment by shlomizadok** [comment]
@ares , changed the test & squashed.

**Comment by ares** [comment]
Thank you, merging

**Pull request merged**